.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
I am not used to seeing linthtmlrc.json here...

Before you push with "git push origin develop"
You add using git add [something here]

In these tasks and in general you only push the code you write or added. So

git add src
- would only add the code you wrote in the src folder

git add readme.md
- would only add the changes in the readme file

it seems like you do

git add .
- which adds everything, kinda inefficient this way.

Let me know if this helps

Reply:
so 

git add src && git add readme.md

should be the right way